### ISSUE TITLE
Success stories page

### ISSUE THREAD

**Issue by svenfuchs** [rails-girls-summer-of-code/summer-of-code-2014#167]
I think we should build on @berlintam's great blog post http://railsgirlssummerofcode.org/blog/where-to-start and create a page `/success-stories/2013`

This actually could just copy a good part of the blog post and make it a static page. We could then also ask other teams to add themselves to it, ask projects to provide their perspective and maybe a few quotes (e.g. @rkh's quote about how impressed he was).

We could also add info about who found a position as employees or interns.

WDYT?

**Comment by sareg0** [comment]
+1 
I think this is a great idea, kind of a Rails Girls 2013 participants; where are they now ;)

This could also give us up to 15 or so articles :) for the blog.

**Comment by sareg0** [comment]
Shall we send those people from last year an email, and direct them to the page, so they can contribute directly. 
Have we sent an email to all of last year's participants yet? We could just email them again.

**Comment by sareg0** [comment]
After a lengthy 

Here is the text we could send to ask for the stories.

Subject: Summer of Code Needs Your Story!

Since your participation in Rails Girls Summer of Code 2013, you have all gone on to amazing and interesting things!

We want you to tell us your stories. What have you been doing since Summer of Code? What did you like about the program? Did it help you change careers, jobs... You life?

Your stories will be used on the Rails Girls Summer of Code website, so that new batches can read your inspirational and varied stories!

Love,
Your Rails Girls Team

--end--

alternative: we write them all individually.

**Comment by anikalindtner** [comment]
ohh such a nice text <3 i'd vote for approaching them individually, because this way you'll get way more answers and maybe even set up a chat for a quick interview or sth? that might help to get a story together easier than if we'd ask them to send us their texts. alternatively we could send them questions, if nobody has time to chat. just an idea... wdyt?

**Comment by sareg0** [comment]
yes, that's is a very good idea. Sending them prepared questions, or just getting them on the phone would be the best thing. Getting a response otherwise will be a bit of trial.

**Comment by sareg0** [comment]
This has sat idle for a little while. I will add the idea to the editorial calendar, to get a deadline of some description on it.
